Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -394,3 +395,49 @@ extension SPAppDelegate {
         EditorFactory.shared.scrollPositionCache.cleanup(keeping: allIdentifiers)
     }
 }
+
+// MARK: - Background Fetch
+//
+@available(iOS 13.0, *)
+extension SPAppDelegate {
+    @objc
+    func registerBackgroundRefreshTask() {
+        BGTaskScheduler.shared.register(forTaskWithIdentifier: BackgroundRefreshConstants.identifier, using: .main) { task in
+            guard let task = task as? BGAppRefreshTask else {
+                return
+            }
+            self.handleAppRefresh(task: task)
+        }
+    }
+
+    private func handleAppRefresh(task: BGAppRefreshTask) {
+        scheduleAppRefresh()
+        print("Ran handle app refersh")
+        simperium.backgroundFetch { result in
+            if result == .failed {
+                NSLog("<< Background Fetch: New Data Received")
+                task.setTaskCompleted(success: false)
+                return
+            } else if result == .noData {
+                NSLog("<< Background Fetch: No Data Received")
+            }
+            task.setTaskCompleted(success: true)
+        }
```

Comment:
Thoughts:
* We should handle expiration of the task. `task.expirationHandler = { ... }`. Not sure if we can/should tell simperium to stop it's networking.
* When the app is waken up and you have a valid token, simperium library automatically connects to simperium and  starts syncing. Here we trigger another sync, which I think may result in 2 sync processes being run at the same time and possibly data corruption? Maybe we can "ride-along" somehow on the normal sync process and add some sort of notification when we think everything is synced 🤔 

/cc @jleandroperez